### PR TITLE
Fix testing against Django 2.0.

### PR DESCRIPTION
**Pull request by mbaechtold**
See https://github.com/jsocol/django-ratelimit/pull/129#discussion_r148491945 for more information.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -12,5 +12,5 @@ deps =
     django19: Django>=1.9,<1.10
     django110: Django>=1.10,<1.11
     django111: Django>=1.11,<1.12
-    django20b1: Django>=2.0b1,<2.1
+    django20: Django>=2.0,<2.1
```
Comment (adamchainz) [inline tox.ini]: this still needs to read `>=2.0b1` or pip won't find it

Comment (mbaechtold) [reply tox.ini]: Ok, I changed it...

File: .travis.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ env:
  - DJANGO_VERSION=1.9
  - DJANGO_VERSION=1.10
  - DJANGO_VERSION=1.11
- - DJANGO_VERSION=2.0b1
```
Comment (jsocol) [inline .travis.yml]: i think this needs to change back, too. the only thing that should've changed was the classifier, afaict

Comment (adamchainz) [reply .travis.yml]: It's fine renaming the test environments, it means less work in the future

Comment (jsocol) [reply .travis.yml]: That's fair, but the version we're installing still needs to be `2.0b1` for now, I think

Comment (adamchainz) [reply .travis.yml]: `Django>=2.0b1,<2.1` in tox.ini is doing that, no?

Comment (adamchainz) [reply .travis.yml]: oh no I see the logic is duplicated in travis file

Comment (jsocol) [reply .travis.yml]: Yeah, Travis isn't using tox directly, so the error it's returning is
```
pip install -q "Django>=${DJANGO_VERSION},<${DJANGO_VERSION}.99" flake8
  Could not find a version that satisfies the requirement Django<2.0.99,>=2.0
```

Comment (mbaechtold) [reply .travis.yml]: I changed the version back. Now all build jobs on Travis are green.

**Pull request merged**